governance/polls/Investigate Offboarding of Multiple Vault Types - June 21, 2021

```diff
@@ -0,0 +1,55 @@
+---
+title: Investigate Offboarding of Multiple Vault Types - June 21, 2021
+summary: Signal your support or opposition to investigate offboarding of multiple vault types.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboard-unprofitable-collateral-types/8628
+vote_type: Plurality Voting
+categories:
+   - Collateral
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Investigate on Offboarding of multiple collateral types - June 21, 2021
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, June 21 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the Mandated Actors should investigate a plan to offboard the following vault types:
+
+* USDT-A
+* UNIV2-AAVEETH-A
```

Comment:
Technically I think these are UNIV2AAVEETH-A and UNIV2LINKETH-A. Maybe doesn't matter.